Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Analytic Rules/IoTDenialofService.yaml

```diff
@@ -41,7 +41,8 @@ query: |
     Entities,
     VendorOriginalId,
     AlertLink,
-    AlertManagementUri
+    AlertManagementUri,
+    Techniques
```

Comment:
Hey @amitcohen25, the kql validation is failing as the name 'Techniques' does not refer to any known column, table. Please check and resolve it thanks.

Reply:
hi @v-sabiraj  I’m not sure why the KQL validation is failing: The “Techniques” column exists in a known column in our Log Analytic – Security Alert table. 
For example:
**SecurityAlert
| where SystemAlertId in("90f328ce-d854-a7cd-ca37-31619942848b") 
| project Techniques**